common/static/common/templates/discussion/new-post.underscore

```diff
@@ -29,6 +29,13 @@
             </div>
         </label>
     </div>
+    <% } else if (is_discussion_division_enabled) { %>
+    <div class="post-field group-visibility">
+        <label class="field-label">
+            <!-- Wrapper for the visibility message filled out by a separate template -->
```

Comment:
Braden seems to have asked about the `div` element, while you put the comment on `span` element.  However, I don't think Adolfo's answer make sense, since the `div` is actually _not_ used in the way he described.  The `div` is needed to be consistent with the other elements in the form, and possibly for styling, so I'm fine with leaving everything as it is (including your comment).